db/seeds.rb

```diff
@@ -1,34 +1,37 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+# frozen_string_literal: true
+
+# This file should contain all the record creation needed to seed the database
+# with its default values. The data can then be loaded with the rake db:seed
+# (or created alongside the db with db:setup).
 require 'yaml'
 
 # We need mail_configuration to create a user account, because it sends a confirmation email.
 MailConfiguration.apply!
 
-puts "[db:seed] Seeding Roles"
-Spree::Role.where(:name => "admin").first_or_create
-Spree::Role.where(:name => "user").first_or_create
+Rails.logger.debug "[db:seed] Seeding Roles"
+Spree::Role.where(name: "admin").first_or_create
+Spree::Role.where(name: "user").first_or_create
```

Comment:
I thought we got rid of `"user"`, or did we just talk about it..

Reply:
The user role is unused at the moment. And we talked about getting rid of roles. But it's still there.